apps/funke/app/crypto/bPrime.ts

```diff
@@ -0,0 +1,34 @@
+import type { AgentContext } from '@credo-ts/core'
```

Comment:
Everything within app is seen as routes, so I think we should add all this logic outside of `app` dir

Reply:
I'll move it out, but now we just get a very messy root where `ios`, `node_modules`, `app` and now `crypto` is defined...

Reply:
We can add a src directory